src/devices/tuya.ts

```diff
@@ -529,13 +529,22 @@ const definitions: Definition[] = [
         exposes: [e.gas(), e.tamper()],
     },
     {
+        fingerprint: [{modelID: 'TS0205', manufacturerName: '_TZ3210_up3pngle'}],
```

Comment:
This is not needed, `zigbeeModel: ['TS0205'],` already makes sure all gets detected as supported.